samples/hybrid-connections/java/simple-http-demo/.gitignore

```diff
@@ -0,0 +1,53 @@
+*.class
+
+#External libs
+extlib/
+
+# Auth files
+*.auth
+*.azureauth
+
+# Local checkstyle
+*.checkstyle
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
+
+# Azure Tooling #
+node_modules
+packages
+
+# Eclipse #
+*.pydevproject
+.project
+.metadata
+bin/**
+tmp/**
+tmp/**/*
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.classpath
+.settings/
+.loadpath
```

Comment:
I'm surprised this is in a .gitignore file in the current directory instead of the .gitignore for the whole repo: [https://github.com/Azure/azure-relay/blob/master/.gitignore](https://github.com/Azure/azure-relay/blob/master/.gitignore)

Did you choose to not simply include the #Eclipse# stuff in the root .gitignore file?